### ISSUE TITLE
Upload "What is Signalen?" video

### ISSUE THREAD

**Issue by AlbaRoza** [Signalen/signalen.org#33]
V2 of the video. Audio corrections done.

**Comment by Ainali** [comment]
Is this one blocked by there not being a Youtube (or similar) account for Signalen yet?

**Comment by AlbaRoza** [comment]
> Is this one blocked by there not being a Youtube (or similar) account for Signalen yet?

I've just solved it! Video up and running here: https://youtu.be/I2Z-mRFt3pg

**Comment by Ainali** [comment]
Nice! Can we upload the subtitles through youtube as well? It could increase discoverability a bit. 

And perhaps it's for another issue, but I think we could make the logo a bit smaller so that it fits in the circle Youtube crops with.

**Comment by felixfaassen** [comment]
Video is live on website
